Client/src/modules/core/components/contact/ContactPage.jsx

```
@@ -0,0 +1,25 @@
+import ContactInfo from "./ContactInfo.jsx";
+import ContactForm from "./ContactForm.jsx";
+import FAQ from "./FAQ.jsx";
+import Map from "./Map.jsx";
```

Biome

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)